### PR TITLE
Update section question

### PR DESCRIPTION
**Pull request by jinety**

### CODE REVIEW

File: practice-two/src/index.html [hunk 1]
```diff
@@ -9,45 +9,46 @@
...
-              <a href="javascript:void(0);" class="menu-item-link">Resources</a>
-            </li>
-          </ul>
-          <button class="btn mr-24" type="button">Sign In</button>
-          <button class="btn mr-24" type="button">Sign Up</button>
-        </nav>
+      <div class="branch-container mr-24">
+        <a href="javascript:void(0);" class="branch-logo">
+          <div class="branch-img">
+            <img src="./assets/logo.png" class="logo" alt="logo" />
```
Comment (hieungo-agilityio) [inline practice-two/src/index.html]: remove space.

File: practice-two/src/style/section/question.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  .question-content {
+    padding: 16px 0px 30px;
+  }
+  
+  .answer {
+    padding-top: 16px;
+  } 
+
+ .question {
+  font-weight: var(--font-weight-s);
```
Comment (phuocduong-agilityio) [inline practice-two/src/style/section/question.css]: Missing code indent

File: practice-two/src/style/section/pricing.css [hunk 3]
```diff
@@ -17,7 +41,7 @@
   align-items: center;
   border-radius: 8px;
   width: 30%;
-  margin: 0 auto;
+  /* margin: 0 auto; */
```
Comment (phuocduong-agilityio) [inline practice-two/src/style/section/pricing.css]: Remove unused code

File: practice-two/src/style/section/header.css [hunk 4]
```diff
@@ -1,13 +1,14 @@
 /* START MAIN HEADER */
 .site-header {
+  background: var(--secondary-color-100);
+}
```
Comment (phuocduong-agilityio) [inline practice-two/src/style/section/header.css:4]: Missing empty line

**Pull request merged**